### DISCUSSION TITLE
Addition of Nix and NixOS

### DISCUSSION THREAD

**Discussion by JustAlternate** [shospodarets/awesome-platform-engineering#21] [Ideas]
Hi, do you guys think Nix and NixOS could be a great addition to this repository ?

Nix is a declarative package manager that can be used to make reproducible dev environment.
And NixOS is a Linux distribution that embrace the system as code approach to making reproducible system configuration.

**Comment by shospodarets** [comment]
Sure, please feel free to drop a PR to add, so it will be reviewed
